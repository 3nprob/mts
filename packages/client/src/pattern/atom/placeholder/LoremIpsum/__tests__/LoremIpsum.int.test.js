import { defaultRender } from './LoremIpsum.int.render'

describe('<LoremIpsum />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
