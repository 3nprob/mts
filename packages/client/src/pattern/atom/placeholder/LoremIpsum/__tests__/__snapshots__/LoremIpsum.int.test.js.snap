// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoremIpsum /> Snaphot must match defaultRender 1`] = `
<div
  className=""
>
  LoremIpsum Under construction
</div>
`;
